<launch>

  <arg name="vel_min" default="0.1"/>
  <arg name="vel_max" default="0.6"/>
  
  <arg name="x_min" default="0.3"/>
  <arg name="x_max" default="0.5"/>
  <arg name="y_min" default="-0.7"/>
  <arg name="y_max" default="-0.3"/>
  <arg name="z_min" default="0.22"/>
  <arg name="z_max" default="0.4"/>

  <node name="parameters_manager_node" pkg="ar_conveyor_launch" type="parameters_manager_node" respawn="false" output="screen">
    <param name="vel_min" value="$(arg vel_min)"/>
    <param name="vel_max" value="$(arg vel_max)"/>
  </node>
  
  <node name="target_publisher_node" pkg="ar_conveyor_launch" type="target_publisher_node" respawn="false" output="screen">
    <param name="x_min" value="$(arg x_min)"/>
    <param name="x_max" value="$(arg x_max)"/>
    <param name="y_min" value="$(arg y_min)"/>
    <param name="y_max" value="$(arg y_max)"/>
    <param name="z_min" value="$(arg z_min)"/>
    <param name="z_max" value="$(arg z_max)"/>
  </node>
  
</launch>